Prevents
acl.cc:1225:48: error: no member named 'join' in namespace 'fmt'
when building with fmt 11

--- a/src/acl.cc
+++ b/src/acl.cc
@@ -4,6 +4,7 @@
 // of patent rights can be found in the PATENTS file in the same directory.
 
 #include <fmt/format.h>
+#include <fmt/ranges.h>
 #include <cstring>
 #include <fstream>
 #include <shared_mutex>
